tests/cases/assert.py

```diff
@@ -0,0 +1,2 @@
+def compare_assert(a: int, b: int):
+    assert a == b
```

Comment:
all new cases should include a `main()`, so that CI works.

when I add a `main(): compare_assert(1, 2)` (which should fail), dart, julia, kotlin and nim all pass.  Only rust and go fail.  https://github.com/adsharma/py2many/issues/36 should be re-opened.